.github/workflows/test.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: yarn --frozen-lockfile
+      - run: yarn prettier
```

Comment:
Shouldn’t we leave the files as they are, so that `yarn test` can detect formatting issues?